src/components/Home.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/prop-types */
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Navbar from './Navbar';
+import Europe from './Europ';
+import CountryList from './CountryList';
+import { fetchCountries } from '../Redux/countries';
+
+const Home = () => {
+  const { countries } = useSelector((state) => state.countries);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchCountries());
+  }, []);
+  return (
+    <div className="homeScreen">
+      <Navbar />
+      <Europe />
+      <CountryList data={countries} />
+    </div>
+  );
+};
```

Comment:
- [ ] You have done well implementing the home page, but please notice that as mentioned in the project requirements
:point_down: you are missing two things
![image](https://user-images.githubusercontent.com/97521557/219846313-6325a9b9-f8b7-4b11-b0b9-0c41ea742c0e.png)

1. currently the home page shows list of categories, in your case countries but there is no way to filter them, I suggest you implement a search or select dropdown so a user can filter those categories in your case say by country name.
2. It is required that along with the category name, you display some metrics data currently you have only the country name displayed and no metric data.

Please try to fix these :+1: